design/block-data-mover/block-data-mover.md

```
@@ -0,0 +1,551 @@
+The current design doesn't break anything for plugin data movers.  
+The enhancement in VolumePolicy could also be used for plugin data movers. That is, users could select a plugin data mover through VolumePolicy as same as Velero built-in data movers.  
+
+## Installation
+No change to Installation.  
+
+## Upgrade
+No impacts to Upgrade. The new fields in the CRDs are all optional fields and have backwards compatible values.  
+
+## CLI
+Backup type parameter is added to Velero CLI as below:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)